code/modules/clothing/head/soft_caps.dm

```diff
@@ -82,6 +82,12 @@
 	icon_state = "rainbowsoft"
 	cap_color = "rainbow"
 
+/obj/item/clothing/head/soft/ferret
+	name = "\improper Ferret Heavy Industries trucker hat"
+	desc = "It's a trucker hat. They're becoming collector items since Ferret went under in 2180."
```

Comment:
sentence grammar is a bit weird at the end

suggestion:
```suggestion
	desc = "It's a trucker hat. For some reason, these hats have begun to become collectibles ever since Ferret Industries went under, two years ago."
```

Reply:
Comma at the end should be excluded it’s an awkward pause

Reply:
Either mine needs a comma after items or yours needs to lose both commas. I don’t really care what the description is so long as it gets the point across.

Reply:
im not sure removing both is right but i'm ESL so whatever, done 👍